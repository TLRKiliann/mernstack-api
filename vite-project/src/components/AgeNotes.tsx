import { Component } from 'react'
import { notesType } from '../Models/notesType'

export default class AgeNotes extends Component<notesType> {
  constructor(props:any) {
    super(props)
  }
  render() {
    return(
      <div>
        Hello Age ! {this.props.age}
      </div>
    )
  }
}