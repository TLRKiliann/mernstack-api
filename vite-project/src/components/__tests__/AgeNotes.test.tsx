//import React from "react";
import {beforeEach, describe, expect, test, it, vi} from 'vitest';
import {fireEvent, render, screen, waitFor} from '@testing-library/react';
//import "@testing-library/jest-dom/extend-expect"
//import "@testing-library/jest-dom";
import { act } from 'react-dom/test-utils';
import {create} from 'react-test-renderer';
import AgeNotes from "../AgeNotes.tsx";

test('MatchSnapShot test AgeNotes 1', () => {
	const tree = create(<AgeNotes />)
	expect(tree.toJSON()).toMatchSnapshot();
})