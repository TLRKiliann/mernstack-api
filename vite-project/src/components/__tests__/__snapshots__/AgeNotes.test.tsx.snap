// Vitest Snapshot v1

exports[`MatchSnapShot test AgeNotes 1 1`] = `
<div>
  Hello Age ! 
</div>
`;
